### PR TITLE
Get Scene API

### PR DESCRIPTION
**Pull request by ardila**
Added test for new endpoint
and some eq methods to make that test easier

### CODE REVIEW

File: nucleus/dataset_item.py [hunk 1]
```diff
@@ -122,24 +122,16 @@ def __post_init__(self):
         )
 
     @classmethod
-    def from_json(cls, payload: dict, is_scene=False):
+    def from_json(cls, payload: dict):
```
Comment (song-william) [inline nucleus/dataset_item.py:125]: Is removing this an argument an API breaking change?

File: tests/test_scene.py [hunk 2]
```diff
@@ -260,6 +260,10 @@ def test_scene_upload_sync(dataset):
 
     response = dataset.append(scenes, update=update)
 
+    first_scene = dataset.get_scene(scenes[0].reference_id)
+
+    assert first_scene == scenes[0]
+
```
Comment (song-william) [inline tests/test_scene.py:266]: Consider adding a simple case to ensure that `!=` also works.

**Review by song-william — APPROVED** [review]

**Review by sasha-scale — APPROVED** [review]

File: nucleus/dataset_item.py (hunk 1 above)
Comment (ardila) [reply nucleus/dataset_item.py:125]: This is almost entirely used internally so I don't think so

File: tests/test_scene.py (hunk 2 above)
Comment (ardila) [reply tests/test_scene.py:266]: Done

**Pull request merged**